tests/integration/v3_tls_test.go

```diff
@@ -72,3 +78,57 @@ func testTLSCipherSuites(t *testing.T, valid bool) {
 		t.Fatalf("expected TLS handshake success, got %v", cerr)
 	}
 }
+
+func TestTLSMinMaxVersion(t *testing.T) {
```

Comment:
minor comment, it would be better to refactor this test case using table driven + golang subtest.